pkg/asset/cluster/tfvars.go

```diff
@@ -134,6 +134,7 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+		customEndpointsMap := installConfig.Config.Platform.AWS.FetchCustomEndpointsMap()
```

Comment:
nit: I would not include the type in the name ([Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation))
i.e. something like
`customEndpointsMap` -> `customEndpoints`
`FetchCustomEndpointsMap` -> `FetchCustomEndpoints`

Reply:
Changed the variable names to remove type in the name.